### PR TITLE
Add code for Repo

### PR DESCRIPTION
**Pull request by Hevienz**

### CODE REVIEW

File: pkg/manager/repo/handler.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+func (p *Server) DescribeRepos(ctx context.Context, req *pb.DescribeReposRequest) (*pb.DescribeReposResponse, error) {
+	s := sender.GetSenderFromContext(ctx)
+	logger.Infof("Got sender: %+v", s)
+	logger.Debugf("Got req: %+v", req)
+	var repos []*models.Repo
+	offset := utils.GetOffsetFromRequest(req)
+	limit := utils.GetLimitFromRequest(req)
+
+	fmt.Println(manager.BuildFilterConditions(req, models.RepoColumns...))
```
Comment (zheng1) [inline pkg/manager/repo/handler.go]: We don't use `fmt` to print log, please use `logger.Debugf`

File: pkg/manager/repo/handler.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+	query := p.db.
+		Select(models.RepoColumns...).
+		From(models.RepoTableName).
+		Offset(offset).
+		Limit(limit).
+		Where(manager.BuildFilterConditions(req, models.RepoColumns...))
```
Comment (zheng1) [inline pkg/manager/repo/handler.go]: `BuildFilterConditions` had refactor yesterday, please use the new arguments

File: pkg/manager/repo/handler.go (hunk 1 above)
Comment (Hevienz) [reply pkg/manager/repo/handler.go]: ok

File: pkg/manager/repo/handler.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+	"openpitrix.io/openpitrix/pkg/constants"
+	"openpitrix.io/openpitrix/pkg/db"
+	"openpitrix.io/openpitrix/pkg/logger"
+	"openpitrix.io/openpitrix/pkg/manager"
+	"openpitrix.io/openpitrix/pkg/models"
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/utils"
+	"openpitrix.io/openpitrix/pkg/utils/sender"
+	"fmt"
```
Comment (zheng1) [inline pkg/manager/repo/handler.go]: `import` block should be sorted. Please execute `make fmt-all` before you commit

File: pkg/manager/repo/handler.go (hunk 2 above)
Comment (Hevienz) [reply pkg/manager/repo/handler.go]: ok, I am fixing this.

File: pkg/manager/repo/handler.go (hunk 3 above)
Comment (Hevienz) [reply pkg/manager/repo/handler.go]: ok

**Pull request closed without merging**